fetch.js

```diff
@@ -311,7 +311,13 @@
       xhr.onload = function() {
         var status = (xhr.status === 1223) ? 204 : xhr.status
         if (status < 100 || status > 599) {
-          reject(new TypeError('Network request failed'))
+          var msg = 'Network request failed'
+          if (xhr.responseText) {
+            msg += ' ("' + xhr.responseText + '")'
```

Comment:
What will responseText typically contain exactly? Which browsers did you test this on? Would it be possible to create automated tests? What does native fetch return in Chrome and Firefox?